src/commands/emulators-start.ts

```diff
@@ -17,7 +19,7 @@ module.exports = new Command("emulators:start")
       throw e;
     }
 
-    utils.logLabeledSuccess("emulators", "All emulators started, it is now safe to connect.");
+    EmulatorLogger.forEmulator(Emulators.HUB).logLabeled("SUCCESS", "emulators", "All emulators started, it is now safe to connect.");
```

Comment:
Huh this is interesting but I like the convention!  So HUB is basically also the owner of the controller?

Reply:
Seemed reasonable to me, anything that isn't owned by a specific emulator is owned by hub basically